service_configuration_lib/spark_config.py

```diff
@@ -980,6 +991,50 @@ def _convert_user_spark_opts_value_to_str(user_spark_opts: Mapping[str, Any]) ->
     return output
 
 
+def compute_approx_hourly_cost_dollars(spark_conf, paasta_pool):
+    per_executor_cores = int(spark_conf.get("spark.executor.cores", DEFAULT_EXECUTOR_CORES))
+    max_cores = per_executor_cores * (int(spark_conf.get("spark.executor.instances", DEFAULT_EXECUTOR_INSTANCES)))
+    min_cores = max_cores
+    if "spark.dynamicAllocation.enabled" in spark_conf and spark_conf["spark.dynamicAllocation.enabled"] == "true":
+        max_cores = per_executor_cores * (int(
+            spark_conf.get("spark.dynamicAllocation.maxExecutors", max_cores)
+        ))
+        min_cores = per_executor_cores * (int(
+            spark_conf.get("spark.dynamicAllocation.minExecutors", min_cores)
+        ))
+    if paasta_pool == "batch":
+        cost_factor = spark_constants.get('batch_cost_factor', 0.041)
+    else:
+        cost_factor = spark_constants.get('stable_batch_cost_factor', 0.142)
+
+    min_dollars = round(min_cores * cost_factor, 5)
+    max_dollars = round(max_cores * cost_factor, 5)
+    if max_dollars*24 > spark_constants.get('high_cost_threshold_daily', 500):
+        log.warning(
+            TextColors.red(
+                TextColors.bold(
+                    f'\n!!! HIGH COST ALERT !!!',
+                )
+            )
+        )
+    if min_dollars != max_dollars:
+        log.warning(
+            TextColors.magenta(
+                f'\nThe requested resources are expected to cost between $ {TextColors.bold(str(min_dollars))} → '
```

Comment:
Change this to just highlight, "Up-to {max} cost per hour. And per day.".  No need for range.